doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -620,6 +620,11 @@ The association of a single fixpoint and a local definition have a special
 syntax: :n:`let fix @ident {* @binder } := @term in` stands for
 :n:`let @ident := fix @ident {* @binder } := @term in`. The same applies for co-fixpoints.
 
+The :n:`wf` and :n:`measure` clauses are Gallina extensions.  They can't be used in
+commands that are processed by the |Coq| kernel, which doesn't support Gallina extensions.
+
+.. todo what commands don't support them?  I could list some or all of them
```

Comment:
There are no commands that are processed directly by the Coq kernel, so this description is imprecise. But what you can simply write is that the fix construct only supports the `struct` fixannot and that the `wf` and `measure` fixannots are only supported in commands like Function and Program Fixpoint.

Reply:
In fact,

```
Definition foo := fix foo (n : nat) { wf n n } := 0.
```

even raises an anomaly! We should report this if it hasn't been done already (no time to do this right now though).

Reply:
> `wf` and `measure` fixannots are only supported in commands like Function and Program Fixpoint.

Please define `like` precisely for our readers.

Reply:
Filed #11816

Reply:
Well, I think this is an exhaustive list but it could be the case that it isn't true. For instance, it could be the case that this is also used by an external plugin. In practice, Equations (which was the best candidate for a user of this) doesn't, and defines its own `by wf` syntax.